### PR TITLE
Update entity details header.

### PR DESCRIPTION
**Pull request by kadams54**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3478/
Test PASSed.

**Comment by kadams54** [comment]
## QA

Pull up the entity details page. Verify that the header looks like the one at https://cloud.githubusercontent.com/assets/5873394/8907463/5fc27a72-3476-11e5-9c42-aaff407985b0.png.

Note that the "Add with" support selector, deploy command input, and sharing buttons are currently missing and will be added later.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3479/
Test PASSed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3481/
Test FAILed.

File: jujugui/static/gui/src/app/components/entity-header/test-entity-header.js [hunk 1]
```diff
@@ -57,6 +57,7 @@ describe('EntityHeader', function() {
                  'rendered downloads does not match entity downloads');
   });
 
+  /* Test will need to be moved to test-entity-content.js
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/entity-header/test-entity-header.js]: Can you just delete this? When the tags are done in entity content it'll be a different test entirely

**Comment by hatched** [comment]
:+1: With that code removal request
QA :ok:

File: jujugui/static/gui/src/app/components/entity-header/_entity-header.scss [hunk 2]
```diff
@@ -29,90 +25,43 @@
...
-                    display: none;
-                }
+    ul.bullets {
+        li {
+            &:after {
+                color: $warm-grey;
+                content: "\2022";
+                vertical-align: middle;
+                margin: 0 5px;
+            }
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/entity-header/_entity-header.scss]: Spaces between class names.

**Comment by huwshimi** [comment]
A few QA things:
- it looks like the top and bottom padding for the header is too small
- the space between the charm icon and name is a bit small
- the space between the charm icon and By charmers... line is a bit small
- the font size for the By charmers... line is too small
- the padding on the Add to canvas may be a bit small

:+1: with those changes.

**Comment by kadams54** [comment]
Thanks all! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3534/
Test PASSed.

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1209

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3535/
Test PASSed.

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**